Source/WebGPU/WebGPU/Queue.mm

```diff
@@ -258,12 +258,23 @@
         });
     }];
 
+    if (m_commitSharedEvent) {
```

Comment:
This works for the test page but is incorrect in general. The correct thing I think to do is:

* associate the shared event with the render target
* encode the signal event when both textures in the array have all their commands encoded.

That might still be incorrect however, since I don't think there is anything preventing a website from rendering to the same texture multiple times in a single frame.